tests/config/selfupdate.toml

```diff
@@ -5,19 +5,20 @@ cert_source = "file"
 server = "http://localhost:8000"
 
 [storage]
-path = "/persistent/prov_selfupdate"
-uptane_metadata_path = "metadata"
-uptane_private_key_path = "ecukey.pem"
-uptane_public_key_path = "ecukey.pub"
-tls_cacert_path = "ca.pem"
-tls_pkey_path = "pkey.pem"
-tls_clientcert_path = "client.pem"
+path = "/tmp/aktualizr-storage"
+uptane_metadata_path = "/tmp/aktualizr-storage/metadata"
+uptane_private_key_path = "/persistent/prov_selfupdate/ecukey.pem"
+uptane_public_key_path = "/persistent/prov_selfupdate/ecukey.pub"
+tls_cacert_path = "/persistent/prov_selfupdate/ca.pem"
+tls_pkey_path = "/persistent/prov_selfupdate/pkey.pem"
+tls_clientcert_path = "/persistent/prov_selfupdate/client.pem"
```

Comment:
Not that it really matters, but you shouldn't need to specify the paths for each of these if you've already specified it in `path`.

Reply:
The main point is that they come from outside the storage path. Is there a more succinct way to define them?

Reply:
Oh crap, for some reason I read this wrong. Only `uptane_metadata_path` is in `path`, so that's the only one my comment would apply to. You can ignore my comment, sorry!

Reply:
No worries!